src/modules/requirements/InRegionRequirement.ts

```diff
@@ -11,8 +11,12 @@ export default class InRegionRequirement extends Requirement {
     }
 
     public hint(): string {
-        return `You must be in one of the following regions: ${
-            this.regions.join(', ')
-        }`;
+        return this.regions.length > 1
+            ? `You must be in one of the following regions: ${
+                this.regions.map((r) => camelCaseToString(Region[r])).join(', ')
+            }`
+            : `You must be in the ${
+                camelCaseToString(Region[this.regions[0]])
+            } region`;
```

Comment:
Shouldn't this alone work even if you have just one Region?

``` suggestion
        return `You must be in one of the following regions: ${
                this.regions.map((r) => camelCaseToString(Region[r])).join(', ')
            }`
```

Reply:
Yes, but I think saying "one of the following regions" and only listing one thing looks a bit odd